src/main/java/com/tstepnik/planner/service/UserService.java

```diff
@@ -30,6 +30,10 @@ public User getLoggedUser(Principal principal) {
         return userRepository.findByLogin(login).orElseThrow(() -> new EntityNotFoundException("User not found"));
     }
 
+    public User getUser(Long id) {
+        return userRepository.findById(id).get();
```

Comment:
What if there is no value? Why don't you return Optional? Also stop making changes to Users code on this branch. It's not called `feature/task` by mistake.

Reply:
i changed the name for feauture/taks-lombok-mapstruct

Reply:
You cannot change branch name just like this. If its `feature/task` then it stays this way. You only should focus on Task on this branch. Adding lombok and mapstruct was a stretch. Next time I am not going to accept anything that changes code it is not supposed to.